### PR TITLE
Updated getting started docs; mentioned analytic.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by metalmatze** [comment]
I guess we can consider merging this now with the incubation happening. Text lgtm.

**Comment by bwplotka** [comment]
Need to fix links first (:

**Comment by yeya24** [comment]
I guess we still want to have this.

File: docs/getting-started.md [hunk 1]
```diff
@@ -8,18 +8,54 @@ slug: /getting-started.md
...
+List of resources useful if you want to help us improving this project! ❤️ This includes all kinds of contributions: docs,
+code and tools.
+
+### Proposals
+
+The definitive list of current and past proposals that were made in Thanos project lifetime.
+
+Proposals can have 5 Statuses (`.Params.Status`): `accepted`, `complete`, `rejected`, `in-review`, `draft`
+
+Each proposal file should be in the format of `<data>-<name>.md`.
```
Comment (yeya24) [inline docs/getting-started.md:56]: `data` or `date` ?

**Review by yeya24 — COMMENTED** [review]
I think it is good to go if you could do a rebase.

**Pull request closed without merging**